metricbeat/docs/modules/kibana.asciidoc

```diff
@@ -3,10 +3,9 @@ This file is generated! See scripts/docs_collector.py
 ////
 
 [[metricbeat-module-kibana]]
-== kibana Module
-
-This is the kibana Module.
+== Kibana Module
 
+This is the Kibana Module and only tracks the high level metrics. To monitor more Kibana metrics, use our {monitoringdoc}/xpack-monitoring.html[X-Pack monitoring] which is available under a free basic license.
```

Comment:
LGTM with the understanding that we'll flesh out the details of what we mean by "high-level metrics" when this module is more mature.